ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -18,6 +18,18 @@ export const getServerSideProps: GetServerSideProps = async context => {
   );
 
   const { renderer_env: rendererEnv } = context.query as PageDataParams;
+
+  const defaultMetadata = {
+    title: 'BBC World Service – BBC News in 43 Languages Worldwide',
+    description:
+      'Broadcasting trusted BBC News and programmes in 43 languages worldwide - on radio, TV, apps, and digital platforms.',
+    image:
+      'https://static.files.bbci.co.uk/ws/simorgh-assets/public/news/images/metadata/poster-1200x630.png',
+    imageAltText: 'BBC World Service Image',
```

Comment:
As per my comment above 
```suggestion
    imageAltText: 'BBC News World Service',
```